source/common/config/filter_json.cc

```diff
@@ -399,5 +399,36 @@ void FilterJson::translateClientSslAuthFilter(
                                       *proto_config.mutable_ip_white_list());
 }
 
+void FilterJson::translateTcpExtAuthzFilter(
```

Comment:
Do you need v1 configuration support? If not, please drop this as we're trying to deprecate v1.

Reply:
I am happy to remove it. It's useful in ut's though, like [here](https://github.com/colabsaumoh/envoy/blob/4c3e58fd4b221ba61c3d28a824ae618b10e480ce/test/common/filter/ext_authz_test.cc#L45)
What it the recommended way of filling out the protobuf in UT's?

Reply:
There are some examples, e.g. see https://github.com/envoyproxy/envoy/blob/master/test/common/access_log/grpc_access_log_impl_test.cc for an example of a unit test that takes protobuf config. I don't think it's worth keeping just for unit tests, although admittedly a lot of the legacy examples do still use v1 JSON. Protobufs are easy to build programatticaly or via JSON/YAML parsing.